python/lsst/display/firefly/firefly.py

```diff
@@ -88,18 +88,18 @@ def __init__(self, display, verbose=False, host="localhost", port=8080,
         global _fireflyClient
         if not _fireflyClient:
             try:
-                _fireflyClient = firefly_client.FireflyClient("%s:%d" % (host, port),
+                _fireflyClient = firefly_client.FireflyClient("%s" % host,
                                         channel=name, basedir=basedir, **kwargs)
             except Exception as e:
-                raise RuntimeError("Unable to connect websocket %s:%d: %s" % (host, port, e))
-            if (host == "localhost"):
+                raise RuntimeError("Unable to connect websocket %s: %s" % (host, e))
+            if (host[:9] == "localhost"):
```

Comment:
This isn't ever going to evaluate to true now because the protocol is included in the host.  I think it would be better done using `urllib`:
```
from urllib.parse import urlparse
parsed_host = urlparse(host)
if(parsed_host.hostname == "localhost"):
```
This will be both more future proof and readable.